src/pybind/mgr/dashboard/module.py

```diff
@@ -265,14 +263,15 @@ class Module(MgrModule, CherryPyConfig):
 
     MODULE_OPTIONS = [
         Option(name='server_addr', type='str', default='::'),
-        Option(name='server_port', type='int', default=8443),
+        Option(name='server_port', type='int', default=8080),
         Option(name='jwt_token_ttl', type='int', default=28800),
         Option(name='password', type='str', default=''),
         Option(name='url_prefix', type='str', default=''),
         Option(name='username', type='str', default=''),
         Option(name='key_file', type='str', default=''),
         Option(name='crt_file', type='str', default=''),
-        Option(name='ssl', type='bool', default=True)
+        Option(name='ssl', type='bool', default=True),
+        Option(name='ssl_server_port', type='int', default=8443)
```

Comment:
Silly thing but I'd move this up to the `server_port` so that we have related options together.

Reply:
Done